src/app/ui/components/iframe.tsx

```diff
@@ -0,0 +1,32 @@
+//
+//  __          __     _____  _   _ _____ _   _  _____
+//  \ \        / /\   |  __ \| \ | |_   _| \ | |/ ____|
+//   \ \  /\  / /  \  | |__) |  \| | | | |  \| | |  __
+//    \ \/  \/ / /\ \ |  _  /| . ` | | | | . ` | | |_ |
+//     \  /\  / ____ \| | \ \| |\  |_| |_| |\  | |__| |
+//      \/  \/_/    \_\_|  \_\_| \_|_____|_| \_|\_____|
+//
+// The purpose of this iframe is to wrap content from external sources,
+// primarily for use with inscriptions. Iframes are dangerous and we
+// need to be very careful with our use of them.
+//
+// Below, we use the sandbox attribute to limit what they can do, as well as
+// disabling any interaction with pointer events and user selection.
+import { CSSProperties } from 'react';
+
+interface IframeProps {
+  onError(): void;
+  src: string;
+  styles: CSSProperties;
+}
+export function Iframe({ onError, src, styles }: IframeProps) {
+  return (
+    <iframe
+      loading="lazy"
+      onError={onError}
+      sandbox="allow-scripts"
+      src={src}
+      style={{ pointerEvents: 'none', userSelect: 'none', ...styles }}
+    />
```

Comment:
Is `styled.iframe` a thing? This'd allow you to pass props into this component without having to use `style`

Reply:
I will double check this but thought I started trying it.

Reply:
Attempted...

<img width="769" alt="Screenshot 2023-12-15 at 1 34 11 PM" src="https://github.com/leather-wallet/extension/assets/6493321/fca2ea57-982e-4a33-985a-6ad600caf502">